templates/screams/new.hamlet

```diff
@@ -1,5 +1,9 @@
-<h1> Scream into the void!
+<h1>Scream into the void!
 
 <form method=post action=@{NewScreamR} enctype=#{enctype}>
   ^{form}
-  <input type="submit" value="Scream!" />
+  <span #scream-length .valid-scream .valid-tweet>0 #
```

Comment:
Submitting with a 0 length tweet results in a form error since this isn't a `Maybe` value. I think _style_ wise, a zero length post is probably valid, though.